app/domain/logs.rb

```diff
@@ -351,42 +351,42 @@ module AuthnJwt
       )
 
       AddingJwtClaimToValidate = ::Util::TrackableLogMessageClass.new(
-        msg: "Adding JWT claim '{0-claim-name}' to list of mandatory claims to be validated",
+        msg: "Adding JWT claim, '{0-claim-name}' to list of mandatory claims to be validated...",
         code: "CONJ00069D"
       )
 
       CheckingJwtClaimToValidate = ::Util::TrackableLogMessageClass.new(
-        msg: "Checking if JWT claim '{0-claim-name}' is mandatory to validate",
+        msg: "Checking if JWT claimת '{0-claim-name}' is mandatory to validate...",
         code: "CONJ00070D"
       )
 
       FetchingJwtConfigurationValue = ::Util::TrackableLogMessageClass.new(
-        msg: "Fetching jwt signing key uri from '{0-resource-id}'",
+        msg: "Fetching JWT signing key URI from authenticator configuration'{0-resource-id}'...",
```

Comment:
@shulifink  please review

Reply:
This log indicating if we fetching singing key from provider uri or from jwks. This is why the parameter for

Reply:
Fetching JWT signing key URI from authenticator configuration conjur/authn-jwt/some-vendor/jwks-uri

Reply:
fine.